product_attribute_value_adj/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Adjustment on Product Attribute Value',
+    'version': '10.0.1.0.0',
+    'category': 'Generic Modules/Base',
+    'summary': 'Disable the "Duplicate" Button on Product Attribute Value',
+    'author': 'Quartile Limited',
+    'license': 'AGPL-3',
+    'website': 'https://www.quartile.co',
+    'depends': [
+        'product_configurator',
```

Comment:
Should this module depend on `product_configurator` instead of `product`?

Reply:
Yes, because product_configurator is the module that allow user to go to Attribute Value 's  form.

Reply:
@yostashiro  oh wait I think I've made a wrong assumption.